lib/bridge.js

```diff
@@ -312,6 +312,28 @@ Bridge.prototype._customiseAppservice = function() {
 
         this.addAppServicePath({
             method: "GET",
+            path: "/_matrix/app/:ver/thirdparty/location",
+            handler: function(req, res) {
+                if (req.params.ver !== "unstable" || !lookupController.parseLocation) {
+                    res.status(404).send("Cannot GET " + req.path + "\r\n");
```

Comment:
Ah this one. @Kegsay commented on that too in the original functions.
That is attempting to preserve the same error message that `express` generates if it receives a request to a path it has no routes for.